### ISSUE TITLE
[Accessibility], WCAG 1.4.13. Tooltips cannot be closed from the front, reading tooltip with screen readers is uncertain

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1986]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/yso/fi/page?uri=http%3A%2F%2Fwww.yso.fi%2Fonto%2Fyso%2Fp19958

## Description of the problem
Some terms, such as culturally sensitive terms, include a button with a note about the term. The tooltip opens from the keyboard focus or by hovering with the mouse. However, tooltips can't be closed without moving the mouse pointer or focus. 
When the screen reader user uses the arrow keys to move to the button, the screen reader finds the button "more information about this term". However, the additional information tooltip does not yet open from the screen reader's focus. The screen reader user must press enter and then the arrow key again to read the contents of the tooltip.

**Fix**:
 Tooltips must be closable without pointer or keyboard focus movement, eg. with esc-key.

 Screen reader users should not need to activate a button or change navigation mode merely to hear explanatory content. 
Informational content must be announced automatically, not discovered accidentally. 

**How to test:** hover or focus the info button. When the tooltip opens, don’t move the pointer or focus, but press esc key to close the tooltip.

Turn on NVDA. Navigate to info button with tab key. Tooltip text should be read when the tooltip opens.

Severity: moderate
Priority: medium

## Additional information (e.g. screenshots) about the problem

<img width="874" height="294" alt="Image" src="https://github.com/user-attachments/assets/8da97c98-fe39-4207-99ee-9aaf73cd9f2f" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
The issue still exists.
